hbase/Dockerfile

```
@@ -159,5 +159,10 @@ ENV HBASE_CONF_DIR=/stackable/hbase/conf
 ENV HOME=/stackable
 ENV PATH="${PATH}":/stackable/bin:/stackable/hbase/bin
 
+
+USER 0
+RUN ln -s /stackable/jmx/regionserver.yaml /stackable/jmx/region-server.yaml
```

Changed in https://github.com/stackabletech/docker-images/pull/494 and adapted in operator https://github.com/stackabletech/hbase-operator/blame/6f9b4d0b50f72663be2568aea8979f2b86dc8558/rust/crd/src/lib.rs#L360
```suggestion
RUN ln -s /stackable/jmx/regionserver.yaml /stackable/jmx/region-server.yaml
RUN ln -s /stackable/jmx/restserver.yaml /stackable/jmx/rest-server.yaml
```

---

Good catch, thx!